lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -950,14 +950,16 @@ namespace Js
         if (threadContext->IsDisableImplicitCall())
         {
             threadContext->AddImplicitCallFlags(Js::ImplicitCall_External);
+            *value = FALSE;
```

Comment:
nice to have, but no actual bugs here: after adding the explicitcallflag the other data shouldn't be used.